nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -20,6 +20,8 @@
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- `buildGoPackage` has been removed. Use `buildGoModule` instead. See the Go section in the nixpkgs manual for details.
```

Comment:
```suggestion
- `buildGoPackage` has been removed. Use `buildGoModule` instead. See the [Go section in the Nixpkgs manual](TODO) for details.
```

I guess we should link to this directly rather than vaguely referring to some "Go section"?

Reply:
Should the link just point to the unstable manual, or to the upcoming stable version for 25.05?